src/Tribe/Tickets.php

```diff
@@ -4217,6 +4218,24 @@ public function __toString() {
 			return $this->class_name;
 		}
 
+		/**
+		 * Return the URL slug to be used by the provider method.
+		 *
+		 * @since TBD
+		 *
+		 * @return string
+		 */
+		public static function get_provider_url_slug() {
+			/**
+			 * Allow filtering of the tickets_provider slug.
+			 *
+			 * @since TBD
+			 *
+			 * @param string  String for which the slug should be named.
+			 */
+			return apply_filters( 'tribe_tickets_provider_url_slug', 'tickets_provider' );
+		}
```

Comment:
Avoid adding new methods to the `Tickets.php` file, it's already 4k+ lines, move the actual contents of the method into the new template tag we are creating.